CalibratedSensor.cpp

```diff
@@ -1,5 +1,5 @@
 /*
-Fochica� - Forgotten Child in Car Alert - http://fochica.com/
+Fochica™ - Forgotten Child in Car Alert - http://fochica.com/
```

Comment:
This is a change from Ansi Western encoding to a utf8 encoding. Utf8 (without bom/signature) is the new convention for the project (compatible with Arduino IDE and VS flavors). Also unix newlines are preferred but no change will be made to all the files at this time.